src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncBlockBodiesRequestHandler.scala

```diff
@@ -0,0 +1,46 @@
+package io.iohk.ethereum.blockchain.sync
+
+import akka.actor.{Scheduler, Props, ActorRef}
+import akka.util.ByteString
+import io.iohk.ethereum.db.storage.BlockBodiesStorage
+import io.iohk.ethereum.network.p2p.messages.PV62.{BlockBodies, GetBlockBodies}
+
+class FastSyncBlockBodiesRequestHandler(
+    peer: ActorRef,
+    requestedHashes: Seq[ByteString],
+    blockBodiesStorage: BlockBodiesStorage)(implicit scheduler: Scheduler)
+  extends FastSyncRequestHandler[GetBlockBodies, BlockBodies](peer) {
+
+  override val requestMsg = GetBlockBodies(requestedHashes)
+  override val responseMsgCode = BlockBodies.code
+
+  override def handleResponseMsg(blockBodies: BlockBodies): Unit = {
+    (requestedHashes zip blockBodies.bodies).foreach { case (hash, body) =>
+      blockBodiesStorage.put(hash, body)
```

Comment:
we need to double check our DataSources are thread safe now ....I can recall `LevelDB` is but IODB not...will it be better to have a single _handleBodies_ queue to push those changes to storage?